KeenClient/KIODBStore.m

```diff
@@ -127,6 +143,8 @@ - (BOOL)openAndInitDB {
 - (void)closeDB {
     // Free all the prepared statements. This is safe on null pointers.
     if(dbIsOpen) {
+        self.dbQueue = nil;
```

Comment:
The whole checkOpenDb business will be creating a new queue before anything else is dispatched. Setting this to nil might expose anywhere that isn't actually using checkOpenDb by mistake.